x-pack/platform/plugins/shared/fleet/server/services/agents/build_status_runtime_field.ts

```diff
@@ -70,7 +70,7 @@ const _buildInactiveCondition = (opts: {
     })
     .join(' || ');
 
-  return `lastCheckinMillis > 0 && ${field('policy_id')}.size() > 0 && ${policyClauses}`;
+  return `lastCheckinMillis > 0  && ${field('policy_id')}.size() > 0 && (${policyClauses})`;
```

Comment:
@nchaulet is the condition intentionally different than in main? https://github.com/elastic/kibana/pull/211779/files#diff-254672ee94b69f9784dd494532cf311f4197dab080519d36cba0f0b4184fa8a5

Reply:
It is different as that backport PR is not merged yet https://github.com/elastic/kibana/pull/211764 I will update the PR when it's merged